xla/stream_executor/rocm/rocm_driver_wrapper.h

```diff
@@ -104,15 +104,21 @@ namespace wrap {
   __macro(hipGetErrorString)                        \
   __macro(hipGraphAddKernelNode)                    \
   __macro(hipGraphAddChildGraphNode)                \
+  __macro(hipGraphAddMemAllocNode)                  \
   __macro(hipGraphAddMemcpyNode)                    \
   __macro(hipGraphAddMemcpyNode1D)                  \
-  __macro(hipGraphExecChildGraphNodeSetParams)      \
+  __macro(hipGraphAddMemsetNode)                    \
   __macro(hipGraphCreate)                           \
   __macro(hipGraphDebugDotPrint)                    \
   __macro(hipGraphDestroy)                          \
+  __macro(hipGraphExecChildGraphNodeSetParams)      \  
```

Comment:
I think there's extra whitespace on the end of this line

Reply:
To be clear, this would block the automatic merging, because the linter is complaining.

Reply:
thanks for pointing out! fixed